config/2gig/ct32.xml

```diff
@@ -115,9 +117,7 @@
       <Item label="HVAC: Normal, Aux Stages: 2, Aux Setup: Elec, Heat Pump Stages: 2, Cool Stages: 2" value="19005954"/>
       <!-- 0x01,0x2,0x2,0x02,0x02 -->
       <Item label="HVAC: Heat Pump, Aux Stages: 2, Aux Setup: Elec, Heat Pump Stages: 2, Cool Stages: 2" value="35783170"/>
-    
-    
-    <!-- 0x02,0x2,0x2,0x02,0x02 -->
+      <!-- 0x02,0x2,0x2,0x02,0x02 -->
     </Value>
     <Value genre="config" index="3" label="Utility Lock" max="1" min="0" size="1" type="list" units="" value="0">
```

Comment:
The manual says this is write only, but I don't know how I would verify that a GET doesn't work? The OZWCP does not let me refresh individual values.

Reply:
You can add ``` write_only="true"``` to the Value entity to set that up.

Reply:
I'll do that, I verified a SET will be ignored.